src/main/scala/mesosphere/marathon/core/health/impl/HealthCheckActor.scala

```diff
@@ -32,7 +34,7 @@ private[health] class HealthCheckActor(
       app.version,
       healthCheck
     )
-    scheduleNextHealthCheck()
+    scheduleNextHealthCheck(Some(HealthCheck.DefaultFirstHealthCheckAfter))
```

Comment:
shouldn't this be the min(userSpecifiedHealthCheckInterval, defaultFirstHealthCheckAfter)?